### PR TITLE
DNF provider for package module

### LINKED ISSUE #11: Add a DNF provider

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Add dnf provider for package module.

Resolves #11

### CODE REVIEW

File: opsmop/facts/platform.py [hunk 1]
```diff
@@ -125,7 +125,10 @@ def default_package_manager(self):
         if distro == "Darwin":
             from opsmop.providers.package.brew import Brew
             return Brew
-        elif distro in [ 'CentOS Linux', 'Red Hat Linux', 'Amazon Linux', 'Scientific Linux' ]:
+        elif distro == "Fedora":
+            from opsmop.providers.package.dnf import Dnf
+            return Dnf
+        elif distro in ['CentOS Linux', 'Red Hat Linux', 'Amazon Linux', 'Scientific Linux']:
             # Amazon Linux, Scientific Linux, add yourselves here after testing
             # this will need some logic to decide when to use dnf
```
Comment (lachmanfrantisek) [inline opsmop/facts/platform.py]: I am not sure, what other logic needs to be implemented.

Comment (mpdehaan) [reply opsmop/facts/platform.py]: Hi and thanks for this!  I think this is probably fine for now.  At whatever point people start using a dnf-backed CentOS or RHEL, we'll eventually need to have a version check in here, or maybe even something a little simpler, like "if shutil.which('dnf')" ...

**Comment by mpdehaan** [comment]
This looks solid to me, I'll merge this and also add the reference to the provider in https://github.com/opsmop/opsmop-demo/blob/master/module_docs/package.py so it shows up in the online docs.

Thanks very much!

**Comment by lachmanfrantisek** [comment]
Thanks for a quick review!

**Pull request merged**